faq.html

```diff
@@ -155,10 +155,13 @@ <h2 id="general">General</h2>
       <a href="{{page.root}}/img/software-carpentry-banner.png">Software Carpentry logo</a>
       are both trademarked,
       and may only be used with our explicit prior permission.
-      We normally grant this to any training course that covers our
-      <a href="#core-topics">core topics</a>
-      has at least one <a href="{{page.root}}/pages/team.html">certified instructor</a> teaching,
-      and sends us summary information about attendees.
+      We normally grant this to any training course that:
+      <ul> 
+       <li>covers our <a href="#core-topics">core topics</a>, </li>
+       <li>has at least one <a href="{{page.root}}/pages/team.html">certified instructor</a> teaching,</li>
+       <li>runs our standardised pre- and post-workshop questionnaires and provides us with the results,</li>
```

Comment:
On Tue, Mar 17, 2015 at 05:57:28AM -0700, Aleksandra Pawlik wrote:
> Or maybe (for now) say "run, provided by us, standardised
> questionnaires"? @wking - dropping this requirement completely is
> not ideal as we need to collect the data, so we should say upfront,
> that the questionnaires will have to be distributed.

If there's a way that folks can handle the questionnaires while still
remaining a self-organized workshop, that's all I was concerned about.
I don't mind if it's steps for the host to follow to distribute,
collect, and submit the questionnaire, or if we're having admins
donate free time to facilitate some of that, so long as there is some
path open for self-organized workshops.

[1]: https://github.com/numfocus/gsoc/issues/6#issuecomment-80761270